docker/config/local_dev.env

```diff
@@ -85,22 +85,17 @@ processor.command_pathname=/stackwalk/stackwalker
 
 # Set the telemetry bucket name explicitly
 destination.telemetry.bucket_name=telemetry_bucket
+telemetry.bucket_name=telemetry_bucket
 
 # webapp
 # ------
 
 ALLOWED_HOSTS=localhost,webapp
-AWS_HOST=localstack-s3
-AWS_PORT=5000
-AWS_ACCESS_KEY=foo
-AWS_SECRET_ACCESS_KEY=foo
 CACHE_LOCATION=memcached:11211
 DATABASE_URL=postgres://postgres:aPassword@postgresql:5432/breakpad
-ELASTICSEARCH_URLS=http://elasticsearch:9200
 SENTRY_DSN=
 SECRET_KEY=secretkey
 STATSD_HOST=statsd
-TELEMETRY_BUCKET_NAME=telemetry_bucket
```

Comment:
This isn't used in the webapp code.

Reply:
I mispoke. I changed this setting to pull from `destination.telemetry.bucket_name`.